Packs/Base/Scripts/SearchIndicatorRelationships/SearchIndicatorRelationships.py

```diff
@@ -51,36 +51,56 @@ def to_context(relationships: list, verbose: bool) -> List[dict]:
     return context_list
 
 
-def handle_stix_types(entities_types):
+def handle_stix_types(entities_types: str) -> str:
     entities_types = argToList(entities_types)
     for e_type in entities_types:
         FeedIndicatorType.indicator_type_by_server_version(e_type)
     return ','.join(str(x) for x in entities_types).replace(', ', ',')
 
 
+def search_relationships_fromversion_6_6_0(args: dict) -> Optional[List[dict]]:
+    for list_arg in ['entities', 'entityTypes', 'relationshipNames']:
+        args[list_arg] = argToList(args[list_arg]) if args[list_arg] else None
+    res = demisto.searchRelationships({'filter': args})
+    return res.get('data', [])
+
+
+def search_relationships_toversion_6_5_0(args: dict) -> Optional[List[dict]]:
+    res = demisto.executeCommand("searchRelationships", args)
+    if is_error(res[0]):
+        raise Exception("Error in searchRelationships command - {}".format(res[0]["Contents"]))
+    return res[0].get('Contents', {}).get('data', [])
+
+
+def search_relationships(args: dict) -> Optional[List[dict]]:
+    entities = args.get('entities', '')
+    entities_types = args.get('entities_types', '')
+    relationships = args.get('relationships', '')
+    limit = int(args.get('limit', '20'))
+    revoked = argToBoolean(args.get('revoked', 'false'))
+    query = 'revoked:T' if revoked else 'revoked:F'
+    handle_stix_types(entities_types)
+
+    args = {
+        'entities': entities,
+        'entityTypes': entities_types,
+        'relationshipNames': relationships,
+        'size': limit,
+        'query': query
+    }
+    if is_demisto_version_ge('6.6.0'):
```

Comment:
What about version 7.0.0?

Reply:
for 6.6.0 and above it's the same behaviour
https://github.com/demisto/etc/issues/47740#issuecomment-1087924793